spring-cloud-gcp-core/src/main/java/org/springframework/cloud/gcp/core/util/package-info.java

```diff
@@ -0,0 +1,20 @@
+/*
+ * Copyright 2017-2018 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * Util classes for core support for Spring Cloud GCP.
```

Comment:
"Utilities shared across Spring Cloud GCP projects"?
Or "Core util classes for Spring Cloud GCP"?